src/cluster_legacy.c

```diff
@@ -4782,7 +4781,7 @@ void clusterCron(void) {
                 if (clusterNodeIsPrimary(myself) && server.cluster->size == 1) {
                     markNodeAsFailingIfNeeded(node);
                 } else {
-                    serverLog(LL_DEBUG, "*** NODE %.40s possibly failing", node->name);
+                    serverLog(LL_NOTICE, "NODE %.40s possibly failing.", node->name);
```

Comment:
```suggestion
                    serverLog(LL_NOTICE, "NODE %.40s (%s) possibly failing.", node->name, node->human_nodename);
```
It looks like we missed this one.